bitwrk-blender/render_bitwrk/settings.py

```diff
@@ -70,10 +70,6 @@ def register(settings):
             name="Show advanced options",
             description="Enables display of expert settings",
             default=False)
-        settings.bitwrk_client_executable_path = StringProperty(
```

Comment:
Not sure about this option.
 - Removing it could cause compatibility issues (I don't really know the rules, maybe it's safe)
 - Maybe we should keep it and show it when `expert_mode` is True. When starting the client, any non-empty path configured here should override the auto-configured path (but also: only when `expert_mode` is active)

Reply:
I created another branch 'ClientInstallationLegacy' (bad name) to allow the user to set the client daemon path only in expert mode. Leaving this in the property panel somewhat defeats the purpose of this client simplification, so I would go with the branch "ClientInstallation" and remove it entirely. I implemented just in case we can test the UX for both.